src/chapters/jdbc-auth.xml

```diff
@@ -346,38 +435,53 @@ mysql-hostname: localhost
 mysql-port: 3306
 mysql-database: <replaceable>guacamole_db</replaceable>
 mysql-username: <replaceable>guacamole_user</replaceable>
-mysql-password: <replaceable>some_password</replaceable></programlisting>
-                <para>For PostgreSQL, the properties are similar, but with different
-                    prefixes:</para>
-                <informalexample>
-                    <programlisting># PostgreSQL properties
+mysql-password: <replaceable>some_password</replaceable>
+                </programlisting>
+            </informalexample>
+            <para>For PostgreSQL, the properties are similar, but with different prefixes:</para>
+            <informalexample>
+                <programlisting># PostgreSQL properties
 postgresql-hostname: localhost
 postgresql-port: 5432
 postgresql-database: <replaceable>guacamole_db</replaceable>
 postgresql-username: <replaceable>guacamole_user</replaceable>
-postgresql-password: <replaceable>some_password</replaceable></programlisting>
-                </informalexample>
+postgresql-password: <replaceable>some_password</replaceable>
+                </programlisting>
+            </informalexample>
+            <para>The SQL Server properties follow the same format:</para>
+            <informalexample>
+                <programlisting># SQL Server properties
+sqlserver-hostname: localhost
+sqlserver-port: 1433
+sqlserver-database: <replaceable>guacamole_db</replaceable>
+sqlserver-username: <replaceable>guacamole_user</replaceable>
+sqlserver-password: <replaceable>some_password</replaceable>
+sqlserver-driver: microsoft2005
+                </programlisting>
             </informalexample>
             <para>The properties absolutely required by the database authentication extension are
                 relatively few and self-explanatory, describing only how the connection to the
                 database is to be established, and how Guacamole will authenticate when querying the
                 database:</para>
             <informaltable frame="all">
-                <tgroup cols="3">
+                <tgroup cols="4">
                     <colspec colname="c1" colnum="1" colwidth="1*"/>
                     <colspec colname="c2" colnum="2" colwidth="1*"/>
-                    <colspec colname="c3" colnum="3" colwidth="2*"/>
+                    <colspec colname="c3" column="3" colwidth="1*"/>
```

Comment:
The attribute name here should actually be `colnum`, not `column`.

Reply:
Fixed.